qemu/tests/cfg/virtio_fs_share_data.cfg

```
@@ -245,24 +245,25 @@
                     viofs_case_insense_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v CaseInsensitive /d 1 /t REG_DWORD'
         - run_stress:
             no extra_parameters socket_group
+            stress_bs = 4 16 64
```

suggest adding the Unit for future extensions:  stress_bs = 4K 16K 64K 100M 1G 10G
```suggestion
            stress_bs = 4K 16K 64K
```

---

corrected